packages/core/docs/commercetools/changelog/5442.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
why its duplicated?

Reply:
There are 3 changelog files - 2 for `core` (they are unrelated, so I decided to split them into two) and this one for `commercetools`.

Reply:
ah right, i overlooked this part, sorry ;)